src/PowerShellEditorServices.Host/CodeLens/PesterCodeLensProvider.cs

```diff
@@ -54,7 +93,7 @@ public PesterCodeLensProvider(EditorSession editorSession)
                     new ClientCommand(
                         "PowerShell.RunPesterTests",
                         "Run tests",
-                        new object[] { scriptFile.ClientFilePath, false /* No debug */, pesterSymbol.TestName })),
+                        new object[] { scriptFile.ClientFilePath, false /* No debug */, pesterSymbol.TestName, pesterSymbol.ScriptRegion.StartLineNumber, _pesterV4_6_0_OrHigherAvailable })),
```

Comment:
This could be:
```
new object[] { scriptFile.ClientFilePath, false /* No debug */, pesterSymbol.TestName, (_pesterV4_6_0_OrHigherAvailable ? pesterSymbol.ScriptRegion.StartLineNumber : null)})),
```

Reply:
Hmm, the C# compiler needed a bit more help with that, my best solution is
```
 _pesterV4_6_0_OrHigherAvailable ? pesterSymbol.ScriptRegion.StartLineNumber : default(int?);
```